geometry_msgs/Twist twist
geometry_msgs/Pose pose
float32 thrust_scaling
